people/people.yml

```diff
@@ -1691,3 +1691,7 @@
   description: Hardware Team supporting Smartfin since 2023
   level: Project Candidate
   start: 2023
+  name: Josephine Dominguez
```

Comment:
This should be its own array element.
```
- name: Josephine Dominguez
  description: Smartfin, Radio Telemetry Tracking, and FishSense since 2023
```